test/integration/smart_answer_flows/estimate_self_assessment_penalties_test.rb

```diff
@@ -129,7 +129,7 @@ class EstimateSelfAssessmentPenaltiesTest < ActiveSupport::TestCase
   #beginning of quick tests
   context "online return, tax year 2012-13" do
```

Comment:
The copy of this test need to be updated? It would make sense if we change the responses.